src/scripts/main.js

```diff
@@ -1,11 +1,21 @@
 'use strict';
 
 function waitFor(element, eventName) {
-// write your code here
-}
+  return new Promise(resolve => {
+    element.addEventListener(`${eventName}`, () => {
```

Comment:
```suggestion
    element.addEventListener(eventName, () => {
```
Do you really need interpolation here?